src/plugins/inputs/away.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+exports.commands = ["away", "back"];
+
+exports.input = function(network, chan, cmd, args) {
+	if (cmd === "away") {
+		let reason = " ";
+
+		if (args.length > 0) {
```

Comment:
So, why do we need to set away if there is no message? That doesn't seem to be part of the standard.

https://en.wikipedia.org/wiki/List_of_Internet_Relay_Chat_commands#AWAY

I'm not sure how I feel about this breaking change.